staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/kmsv2/envelope.go

```diff
@@ -46,7 +46,7 @@ func init() {
 
 const (
 	// KMSAPIVersion is the version of the KMS API.
-	KMSAPIVersion = "v2alpha1"
+	KMSAPIVersion = "v2beta1"
```

Comment:
going to beta means we are expected to still support beta once we graduate to GA ("v2"). Are we expecting to make everything that looks at this constant start looking at two constants and honor both?

Reply:
Yeah I think so.

Reply:
answering my own question: yes... this would be checked against the encryption config file and the kms plugin status, and could be used to gate/trigger any behavior changes between beta and GA

Reply:
that's right!